arviz/stats/density_utils.py

```diff
@@ -401,80 +401,80 @@ def kde(x, circular=False, **kwargs):
     .. plot::
```

Comment:
There is no empty line between the text aand the `.. plot::` directive so it's not rendering properly: https://arviz-devs.github.io/arviz/api/generated/arviz.kde.html

Reply:
Okay